docs/ecosystem-jasmine-dom.md

```diff
@@ -0,0 +1,35 @@
+---
+id: ecosystem-jasmine-dom
+title: jasmine-dom
+---
+
+[`jasmine-dom`][gh] is a companion library for `React Testing Library` that
+provides custom DOM element matchers for Jasmine
+
+```
+npm install --save-dev @testing-library/jasmine-dom
+```
+
+Then follow [usage section][gh-usage] from jasmine-dom's documentation to add
+the matchers to Jasmine.
+
+```jsx
+<span data-testid="not-empty"><span data-testid="empty"></span></span>
+<div data-testid="visible">Visible Example</div>
+
+expect(queryByTestId(container, 'not-empty')).not.toBeEmptyDOMElement()
+expect(getByText(container, 'Visible Example')).toBeVisible()
```

Comment:
```suggestion
expect(screen.queryByTestId('not-empty')).not.toBeEmptyDOMElement()
expect(screen.getByText('Visible Example')).toBeVisible()
```

Most examples should use `screen` rather than passing a container.